tests/e2e/test_rosacli_cluster.go

```diff
@@ -94,7 +101,17 @@ var _ = Describe("Edit cluster",
 
 				Expect(CD.State).To(Equal(jsonData.DigString("status", "state")))
 				Expect(CD.Created).NotTo(BeEmpty())
-				Expect(CD.DetailsPage).NotTo(BeEmpty())
+
+				By("Get details page console url")
+				consoleURL := common.GetConsoleUrlBasedOnEnv(ocmApi)
+				subscriptionID := jsonData.DigString("subscription", "id")
+				if consoleURL != "" {
+					if ocmApi == constants.StageEnv {
```

Comment:
No meaning to check if condition here now. Both production and staging are sharing same code `Expect(CD.DetailsPage).To(Equal(consoleURL + subscriptionID))`